### ISSUE TITLE
False positive with rxjs-no-subclass

### ISSUE THREAD

**Issue by timbru31** [cartant/rxjs-tslint-rules#87]
Hi there 👋,

thanks for these rules, really awesome 👍  
I'm having a false positive with the rule `rxjs-no-subclass`. OpenLayers defines Observables, too.  
Below is an example snippet which triggers the rule:

```ts
declare module 'ol/Object' {
	import Observable from 'ol/Observable';

	// tslint:disable-next-line:rxjs-no-subclass
	class BaseObject extends Observable {
		get(key: string): any;
	}
}
```

The `Observable` class is:
```ts
declare module 'ol/Observable' {
	export interface GlobalObject { [key: string]: any; }
	export type EventsKey = GlobalObject;

	class Observable {
		constructor();
		changed(): void;
		getRevision(): number;
		on(type: (string | string[]), listener: Function): (EventsKey | EventsKey[]);
		once(type: (string | string[]), listener: Function): (EventsKey | EventsKey[]);
		un(type: (string | string[]), listener: Function): void;
	}

	export default Observable;
}

```

**Comment by cartant** [comment]
This should be solvable. However, it will involve digging around in TypeScript's `Type` representation. And, AFAICT, the documentation is ... the source. I will have a look at this on the weekend. Hopefully, it won't be too much of  a challenge.

**Comment by cartant** [comment]
Should be fixed in 4.18.2.

**Comment by timbru31** [comment]
Nice, thanks a lot for the quick fix. I’ll test it later today
